x-pack/platform/plugins/shared/alerting/common/routes/rule/apis/mute_alert/schemas/v1.ts

```diff
@@ -19,3 +19,16 @@ export const muteAlertParamsSchema = schema.object({
     },
   }),
 });
+
+export const muteAlertQuerySchema = schema.maybe(
+  schema.object({
+    validate_alerts_existence: schema.maybe(
+      schema.boolean({
+        defaultValue: true,
```

Comment:
The existing behavior is essentially, `defaultValue: false`, so we're changing the default behavior.  Specifically, for anyone that "pre-mutes" alert instance id's, they'd have to add the parameter.  

So, feels like we should make the default value `false` instead ...

Reply:
In our public-facing documentation, it’s stated that it should return a 404, so I believe this isn’t a breaking change - it’s actually a bugfix.

Reply:
Hey, @ersin-erdal et. al - couple of follow-up questions below for release notes:

> The release note should mention that we fixed the validation by default. 

With this fix, what's the new default behavior?

> If the user still wants to pre-mute alerts, we should note that the validate_alerts_existence flag is available and that validation can be disabled.

I'm not familiar with the concept of pre-muting alerts. Can someone help me understand what this and how it works?